stencil.config.ts

```diff
@@ -97,6 +97,7 @@ export const create: () => Config = () => ({
     }
   ],
   globalStyle: "src/assets/styles/global.scss",
+  globalScript: "src/utils/theme.ts",
```

Comment:
This can be done separately, but can you introduce a global script? It’ll be clearer without looking at the config and allow us to add additional things if needed.

Reply:
I created `src/utils/index.ts` which is now the file used for the `globalScript` stencil config option. Is that what you meant? 

Is that a good filename and directory? An alternative filename could be `globalScript.ts` and alternative directories could be `support` or `src`.

Reply:
`utils` makes sense to me. I'd suggest renaming to `global` or `globalScript` to make its use clearer.